Docs/Tests.md

```diff
@@ -0,0 +1,157 @@
+# Tests
+
+## Core Test group
+
+Spring 2020 we installed a Core test group in Signal for FN2 (now called 'Gordian Wallet'). See [Contributing](../Readme.md#Contributing).
```

Comment:
Gordian Wallet tests are quite different to FN tests, they prob should not be included